packages/manager/src/managers/telemetry/TelemetryManager.ts

```diff
@@ -272,15 +272,17 @@ export class TelemetryManager extends BaseManager {
 	async checkIsTelemetryEnabled(): Promise<boolean> {
 		let root: string;
 		try {
-			root = await this.project.getRoot();
+			root = await this.project
+				.getRoot()
+				.catch(() => this.project.suggestRoot());
+
+			return readPrismicrc(root).telemetry !== false;
 		} catch {
-			root = await this.project.suggestRoot();
+			return false;
```

Comment:
Dump question, maybe we could default to true? If we didn't find the `prismicrc` with telemetry `false`, then it's true by default. WDYT?

Reply:
No idea I'm not very well versed in GDPR stuff :) but why not!